### ISSUE TITLE
Please add root check

### ISSUE THREAD

**Issue by vay3t** [PowerShellEmpire/Empire#296]
## Empire Version
1.5

## OS Information (Linux flavor, Python version)
Raspbian GNU/Linux 8 (jessie) Python 2.7.9

## Expected behavior and description of the error, including any actions taken immediately prior to the error. The more detail the better.

when i execute the listener i have the error message
```
(Empire) > listeners
[!] No listeners currently active 
(Empire: listeners) > set Name WinLab
(Empire: listeners) > execute
[!] Exception: attempt to write a readonly database
```

and have problem to exit, never end the process (loop)
```
[>] Exit? [y/N] y

[!] Shutting down...
```

## Screenshot of error, embedded text output, or Pastebin link to the error
![captura de pantalla de 2016-08-05 10-30-07](https://cloud.githubusercontent.com/assets/6080369/17439830/d0a5d9de-5af7-11e6-8831-df557f678fbb.png)
![captura de pantalla de 2016-08-05 10-33-14](https://cloud.githubusercontent.com/assets/6080369/17439900/0c617f78-5af8-11e6-8eaa-6ef45f60a2e4.png)

## Fixed
need sudo. Please add root check

**Comment by enigma0x3** [comment]
Good idea, we will get this implemented.

Thanks!

**Comment by HarmJ0y** [comment]
So you shouldn't need sudo unless you created the database/setup under root privileges and then attempt to use Empire as an unprivileged user, is this the case?

**Comment by vay3t** [comment]
for some strange reason I have had many operational problems in empire unused root

**Comment by leesoh** [comment]
I do as well. Will get more info when I get home.

**Comment by leesoh** [comment]
Just re-read @HarmJ0y's message. You're correct.

**Comment by HarmJ0y** [comment]
This will be corrected in the 2.0 rewrite, which will be out in a few weeks at DerbyCon.

**Comment by enigma0x3** [comment]
I am confirming that there are root checks now in the 2.0 version, which @HarmJ0y mentioned will be released at DerbyCon in a few weeks.
